docs/tutorial/part-two/index.md

```diff
@@ -496,6 +496,8 @@ The final Glamor page should look identical to the CSS Modules page.
 
 ![glamor-example](glamor-example.png)
 
+In addition, you can also use this plugin to make [`glamorous`](https://github.com/paypal/glamorous) 💄 work with server side rendering and start writing React components that carry their styles with them.
```

Comment:
Oh sorry, my tweet wasn't specific enough — add the mention of Glamorous to https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-glamor/README.md and then add a Glamorous link along with links to Emotion, JSS, Stylus, etc.